contents/docs/integrate/ingest-live-data.mdx

```diff
@@ -121,4 +124,29 @@ import ReactNativeIdentify from './client/react-native/snippets/identify.mdx'
   <TabPane tab="React Native" key="5">
     <ReactNativeIdentify />
   </TabPane>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+# Event ingestion nuances
+
+It's top-priority for us that events are fully processed and saved as soon as possible. However, there is a class of events which we _deliberately_ process with a slight delay. Specifically, an event is delayed by around a minute if it fits those three conditions:
+
+- isn't anonymous (anonymous events are recognized by having the `distinct_id` the same as the `$device_id` property)
+- isn't an `$identify` event (i.e. `posthog.identify()`)
+- its `distinct_id` cannot be matched to an existing person
+
+This delay mechanism is called the **event buffer**, and it materially improves handling of an edge case which could otherwise inflate unique user counts.
+
+## How does the event buffer help?
+
+Starting with version 1.38.0, PostHog stores the person associated with an event inline with the event record. This greatly improves query performance, but because events are immutable, it also means that persons can't be merged retroactively. See this scenario where that's problematic:
+
+1. User visits signup page, in turn frontend captures anonymous `$pageview` for distinct ID `XYZ` (anonymous distinct ID = device ID).  
+  This event gets person ID `A`.
+2. User click signup button, initiating in a backend request, in turn frontend captures anonymous `$autocapture` (click) for distinct ID `XYZ`.  
+  This event gets person ID `A`.
+3. Signup request is processed in the backend, in turn backend captures identified signup for distinct ID `alice@example`.com.  
+  OOPS! We haven't seen `alice@example.com` before, so this event gets person ID `B`.
+4. Signup request finishes successfully, in turn frontend captures identified $identify aliasing distinct ID `XYZ` to `alice@example.com`.  
+  This event gets person ID `A`.
+
+Here, the event from step 3 got a new person ID `B`, impacting unique users counts. If it were delayed just a bit and processed after the event from step 4, all events would get the expected person ID `A`. This is exactly what the event buffer achieves.
```

Comment:
Unsure we want to share this level of depth yet

Reply:
I think it may be a bit too much detail for this article, but this definitely needs to be explained _somewhere_, so I propose we keep the rationale here _until_ we have a proper article on persons-on-events and implications that are good to be aware of.

Reply:
Put this into an expandable section, so shouldn't be overwhelming now:
<img width="305" alt="Screen Shot 2022-06-30 at 18 27 41" src="https://user-images.githubusercontent.com/4550621/176729483-b38eb5d7-5eff-469f-a4fe-39ec297bfafc.png">